contracts/test/_global-hooks.js

```diff
@@ -34,12 +34,25 @@ mocha.before(function () {
 
   // If you are running unit tests, scrape out all fork tests.
   // For fork tests, scrape out all unit tests.
-  root.suites = root.suites.filter(
-    (s) =>
-      s.file.endsWith(".fork-test.js") == isForkTest &&
-      s.file.endsWith(".arb.fork-test.js") == isArbFork &&
-      s.file.endsWith(".base.fork-test.js") == isBaseFork
-  );
+  root.suites = root.suites.filter((s) => {
+    const isMainnetForkTestFile = s.file.endsWith(".fork-test.js");
+    const isHoleskyTestFile = s.file.endsWith(".holesky-fork-test.js");
+    const isArbTestFile = s.file.endsWith(".arb.fork-test.js");
+    const isBaseTestFile = s.file.endsWith(".base.fork-test.js");
+
+    if (isMainnetForkTest) {
```

Comment:
I think I changed the order and fixed it in some PR. `isMainnetForkTestFile` should always be at the end and should always be fully spread like `isMainnetForkTestFile && !isHoleskeyTestFile && !isArbTestFile`

Reply:
Thanks for the fix